app/src/test/java/org/simple/clinic/contactpatient/ContactPatientEffectHandlerTest.kt

```diff
@@ -33,12 +34,22 @@ class ContactPatientEffectHandlerTest {
 
   private val clock = TestUserClock(LocalDate.parse("2018-01-01"))
 
+  private val facility = TestData.facility(
+      uuid = UUID.fromString("251deca2-d219-4863-80fc-e7d48cb22b1b"),
+      name = "PHC Obvious",
+      facilityConfig = FacilityConfig(
+          diabetesManagementEnabled = true,
+          teleconsultationEnabled = false
+      )
+  )
+
   private val effectHandler = ContactPatientEffectHandler(
       patientRepository = patientRepository,
       appointmentRepository = appointmentRepository,
       clock = clock,
       schedulers = TrampolineSchedulersProvider(),
-      uiActions = uiActions
+      uiActions = uiActions,
+      currentFacility = { facility }
```

Comment:
Ideally this change should be done in the commit where we are changing `Init` block